db/post_migrate/20240829140226_drop_old_notification_id_columns.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class DropOldNotificationIdColumns < ActiveRecord::Migration[7.1]
+  DROPPED_COLUMNS ||= {
```

Comment:
```suggestion
  DROPPED_COLUMNS = {
```

Reply:
All migrations except one use `||=` rather than `=`. In this case, I used it for consistency purposes. How strongly do you feel about this?

Reply:
Quite strongly because all the other migrations are doing something wrong 😅  A constant is never defined twice. If it is, it is a bug.